packages/browser/src/index.ts

```diff
@@ -1,377 +1,76 @@
-import {
-  CodeTokenResponse,
-  createRequester,
-  decodeIdToken,
-  fetchOidcConfig,
-  fetchTokenByAuthorizationCode,
-  fetchTokenByRefreshToken,
-  generateCodeChallenge,
-  generateCodeVerifier,
-  generateSignInUri,
-  generateSignOutUri,
-  generateState,
-  IdTokenClaims,
-  Prompt,
-  Requester,
-  revoke,
-  verifyAndParseCodeFromCallbackUri,
-  verifyIdToken,
-  withReservedScopes,
-} from '@logto/js';
+import BasicClient, {
```

Comment:
nit: Basic -> Base
`BaseClient` sounds better to me. Feels like something you should extend before use. But `BasicClient` sounds like a minimal usable client.

Reply:
updated